tests/tests/da_compression.rs

```diff
@@ -68,7 +110,19 @@ async fn can_fetch_da_compressed_block_from_graphql() {
 
     let block = client.da_compressed_block(block_height).await.unwrap();
     let block = block.expect("Unable to get compressed block");
-    let _: VersionedCompressedBlock = postcard::from_bytes(&block).unwrap();
+    let block: VersionedCompressedBlock = postcard::from_bytes(&block).unwrap();
+
+    let db = &srv.shared.database;
+    let mut tx_inner = db.off_chain().clone().into_transaction();
+    let db_tx = DecompressDbTx {
+        db_tx: DbTx {
+            db_tx: &mut tx_inner,
+        },
+        onchain_db: db.on_chain().view_at(&0u32.into()).unwrap(),
+    };
+    let decompressed = decompress(compression_config, db_tx, block).await.unwrap();
```

Comment:
I guess I'm a little confused by using the shared db for decompressing the info. Shouldn't we be able to decompress it with an isolated, clean db?

Maybe that's not the point of this test.

Reply:
decompression requires some previous indexation to be done on the data, it can't be done from an empty database afaiu